android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java

```diff
@@ -133,6 +133,7 @@ public void configure(NativeViewGestureHandler handler, ReadableMap config) {
     @Override
     public void extractEventData(NativeViewGestureHandler handler, WritableMap eventData) {
       eventData.putBoolean("pointerInside", handler.isWithinBounds());
+      eventData.putDouble("numberOfTouches", handler.getMaxNumberOfPointers());
```

Comment:
Not sure if it was intended to provide a max number of pointers here but my intuition for this particular event param would be that it represents the current number of touches (much like pointerInside refers to the current state of the handler).
Also consider the edge case when this method is used for the state change event and make sure this is consistent with the behaviour on iOS.